index.html

```diff
@@ -1232,9 +1232,7 @@ <h2>Definition of Roles</h2>
           </tr>
           <tr>
             <th class="role-base-head" scope="row">Base Concept:</th>
-            <td class="role-base">
-                <a href="https://www.w3.org/TR/html5/grouping-content.html#the-dl-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>dl</code></a>
-            </td>
+            <td class="role-base"><code>&lt;[^dl^]&gt;</code> in HTML</td>
```

Comment:
I settled on a syntax convention of “`<element-name> in HTML`”, where just the element name is linked using its xref term.  I kept (or added, where needed) the `&lt;` and `&gt;` brackets to aid in quick understandability/differentiation when referencing an **HTML element** versus a plain text role string or attribute name.